auto-merge.yml

```diff
@@ -0,0 +1,4 @@
+min-approvals:
+  COLLABORATOR: 1
```

Comment:
I would be more confortable with `member`

Reply:
Done, you're right, it's better.